frameworks/kafka/docs/security.md

```diff
@@ -105,10 +72,8 @@ For example, if installing with these options:
             "realm": "EXAMPLE"
```

Comment:
The "kerberos" section is not nested in the `service.security` section.

Reply:
Yep I noticed this when working with Richard... will fix

Reply:
Ok, it's fixed here, but not for HDFS.